packages/terra-tag/src/Tag.jsx

```diff
@@ -59,6 +59,10 @@ class Tag extends React.Component {
     // Apply focus styles for keyboard navigation
     if (event.nativeEvent.keyCode === KeyCode.KEY_TAB) {
       this.setState({ focused: true });
+    }else if (event.nativeEvent.keyCode === KeyCode.KEY_RIGHT) {
+      this.setState({ focused: true });
+    }else if (event.nativeEvent.keyCode === KeyCode.KEY_LEFT) {
+      this.setState({ focused: true });
```

Comment:
```suggestion
    }else if (event.nativeEvent.keyCode === KeyCode.KEY_RIGHT) {
      this.setState({ focused: true });
    }else if (event.nativeEvent.keyCode === KeyCode.KEY_LEFT) {
      this.setState({ focused: true });
```
Can we move this to list component as tag will support arrow keys only when they are used in group and not when they are used as individual element

Reply:
there were some issues seen with the control flow when I had tried this before. If it's a necessity, will try and implement it. Should I?